sdk/graphservices/Azure.ResourceManager.GraphServices/tests/GraphServicesManagementTestBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.GraphServices.Tests
+{
+    public class GraphServicesManagementTestBase : ManagementRecordedTestBase<GraphServicesManagementTestEnvironment>
+    {
+        protected ArmClient Client { get; private set; }
+
+        protected GraphServicesManagementTestBase(bool isAsync, RecordedTestMode mode)
+        : base(isAsync, mode)
+        {
+        }
+
+        protected GraphServicesManagementTestBase(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        public void CreateCommonClient()
```

Comment:
we could just add a [SetUp] attribute on this method, and use the property `Client` in the test cases.